### DISCUSSION TITLE
Can't import public repository

### DISCUSSION THREAD

**Discussion by Dripcoding** [vercel/vercel#5918] [Help]
Hi Vercel,

I'm trying to deploy my public [github repository](https://github.com/Dripcoding/front-end-mentor-social-proof) from a personal account but am unable to import it to create a new project. 

The repository I want to deploy doesn't appear in the list of available repos. 

![image](https://user-images.githubusercontent.com/65139698/109446706-04d6c880-79f7-11eb-93d9-6ad96a33a926.png)

**Comment by Dripcoding** [comment]
I was able to deploy my project using the vercel CLI tool but I can't link the project to the repository in the web app.

**Reply by nblthree** [reply]
In GitHub go to setting -> applications -> configure Vercel -> Repository access

**Reply by Dripcoding** [reply]
@nblthree Thanks! That did the trick.
